src/styles/style.css

```diff
@@ -0,0 +1,24 @@
+* {
```

Comment:
Consider separating the styles for each BEM block into individual files as per the task requirements. This will help in maintaining a modular and organized codebase.